### ISSUE TITLE
Project Maturity

### ISSUE THREAD

**Issue by MartinThoma** [eoranged/rq-dashboard#404]
I see that the last commit is from 2020 and the readme contains the consent

> The RQ dashboard is currently being developed and is in beta stage.

Is the project dead?

**Comment by ThomasThelen** [comment]
Also curious about this. There are 26 open pull requests, some of which allow for updating dependent libraries which I'd love to incorporate. It looks like the two major contributors from Parallels may not have been active on GitHub for some time. I hope I'm not out of line by pinging @alevlasu - who works on another [cool package](https://github.com/Parallels/vagrant-parallels) from Parallels. Maybe they can have a word about the status/plan here?
I'm sure it would be possible to find a community moderator to help keep the repo going if necessary!

**Comment by sbrady** [comment]
@MartinThoma I think you might be correct.
maybe we should rally around some fork?

**Comment by cjlapao** [comment]
@MartinThoma, @ThomasThelen, @sbrady 
From Parallels side, this is no longer actively being developed, but looking at the sheer number of users using it, I will review the open PRs and merge what makes sense.
I will also try to move this to an open-source project, this way we can allow myself and others to push it further.

Let me know what would be a good improvement to keep it alive?

**Comment by sbrady** [comment]
It is probably important to note that: https://python-rq.org/docs/monitoring/ actually recommend this project.

@cjlapao Maybe give someone from the community access?

Thank you, I think the PRs with library bumps are the most important

**Comment by cjlapao** [comment]
@sbrady 

I will see how we might open this to the community, but for now, I am reviewing all of the open PRs, I have already merged some of the ones done by users on the bump. I will try out the ones done by Dependabot as not sure if they break the code.

Hopefully I will get this done soon

**Comment by cjlapao** [comment]
@sbrady @ThomasThelen 

I have now closed all the prs for dependency upgrades and added the fixes for these dependencies to work, while I do not use this, running the project in docker works fine to connect to a redis server.

Next step would be to deal  with the other users pull requests one by one but this might take time

We will also try to move the repo to a more modern one, by adding some github actions to build this

Will keep you posted

**Comment by ThomasThelen** [comment]
@cjlapao thanks a million. I understand that maintaining open source projects from the past can be a time consuming challenge-and we all appreciate the work you've put in here! If anything can be done on the project management side of things or if there are specific PR's that you like to be tested - feel free to reach out.

**Comment by sbrady** [comment]
thanks @cjlapao

**Comment by ajnisbet** [comment]
@cjlapao you could consider reaching out to [jazzband](https://github.com/jazzband): they are a group which adopts and maintains widely-used python packages that need more maintainers. It seems like rq-dashboard would meet their [guidelines](https://jazzband.co/about/guidelines). I'd be happy to help with any work needed to bring the repo up to their requirements.

(And thanks for staying engaged with this project so far down the line!)

**Comment by cjlapao** [comment]
@ajnisbet , unfortunately, they require the project transfer, which we cannot do now.
The plan, for now, is to start a proper community and move it from there. I am creating a channel on our Discord server for this project and configuring the CI/CD pipelines.

This will allow us as a community to improve, update, and keep it moving forward.

@sbrady , @ThomasThelen, the help would be appreciated for validating of the current open PRs and if they are still of importance after the updates. If any of you could do that, it would be a massive help.

We also are setting up a project with a kanban dashboard to add the new features and work on issues

**Comment by ThomasThelen** [comment]
I'll take a look at the unapproved PR's:
1. #350 
2. #346 
3. #376

**Comment by cjlapao** [comment]
I have now finalized the automation of this project. we will be able to create releases very easily
Added a discussion page and pr issue templates.

Now on the testing side they don't seem to work. I have opened a discussion looking for volunteers to try to fix it, so the building process is ignoring the failures

I also added a channel to Parallels Discord server for discussion.

@sbrady @ThomasThelen what do you think of closing this issue and moving the project maturity and future addons to the discussion or discord place?

**Comment by cjlapao** [comment]
@ThomasThelen did you had the change to have a look at those pr's?

**Comment by ThomasThelen** [comment]
Oops sorry about that! Reviewed.

**Comment by ThomasThelen** [comment]
Given that rq-dashboard has remained active since this was opened 4 years ago, maybe we can close this issue?

**Comment by cjlapao** [comment]
@ThomasThelen yes, it is under maintenance so not a lot new things are active but we will keep the lights on for now
